interface/forms/vitals/templates/vitals/vitals.html.twig

```diff
@@ -14,10 +14,10 @@
 <script src="{{ FORM_ACTION|attr }}/interface/forms/vitals/vitals.js?v={{ assetVersion|attr_url }}" type="text/javascript"></script>
 <script>
     let vitalsTranslations = {
-        'weight_input': window.xl("Weight") + ' (' + window.xl("lbs") + ')'
+        'weight_input_usa': window.xl("Weight") + ' (' + window.xl("lbs") + ')'
         ,'weight_input_metric': window.xl("Weight") + ' (' + window.xl("kg") + ')'
-        ,'height_input': window.xl("Height/Length") + ' (' + window.xl("in") + ')'
-        ,'height_input_metric': window.xl("Height/Length") + ' (' + window.xl("cm") + ')'
+        ,'height_input_usa': window.xl("Height/Length") + ' (' + window.xl("in") + ')'
+        ,'height_input_usa_metric': window.xl("Height/Length") + ' (' + window.xl("cm") + ')'
```

Comment:
This has both usa and metric in the name?

Reply:
thanks